src/api/supportRequest.ts

```diff
@@ -0,0 +1,60 @@
+import express from 'express';
+import { adminMiddleware } from './adminMiddleware';
+import { SupportRequest, SupportRequestStatus, SupportRequestType } from '../entities/supportRequest';
+import logger from '../logger';
+import messageUsers from '../slack/utilities/messageUsers';
+
+export const supportRequestRoutes = express.Router();
+supportRequestRoutes.use(adminMiddleware);
+
+supportRequestRoutes.post('/getNext', async (req, res) => {
+  let nextRequest;
+  try {
+    nextRequest = await SupportRequest.getNextSupportRequest();
+    res.send(nextRequest);
+  } catch (err) {
+    logger.error('Something went wrong trying to get the next support request', err);
+    return;
+  }
+
+  try {
+    await messageUsers(
+      [nextRequest.slackId],
+      `:tada: We're ready to ${
+        nextRequest.type === SupportRequestType.IdeaPitch ? 'help you with an idea' : 'help with your technical issue'
+      }, so head over to our booth. Feel free to bring other members of your team and make sure to bring your laptop if relevant.`,
+    );
+  } catch (err) {
+    logger.error("Unable to notify users they're support request has been served", err);
+  }
```

Comment:
Can we wrap this in a `promise` or an `async` function and not `await` it so that the api call isn't waiting for this to resolve before returning?

Reply:
The API call is acknowledged up on line 13, this is after the request goes out. The only other thing I could do would be to create a new interface and add the `supportRequest` to it along with a `userNotified` bool. Thoughts? If not, wrapping this in a promise won't change anything.

Reply:
AFAIK it doesn’t actually send the response until it returns though

Reply:
Might as well make the interface then. Cool?